rbac/management/tasks.py

```diff
@@ -50,3 +50,9 @@ def run_reconcile_tenant_relations_in_worker(kwargs):
 def run_sync_schemas_in_worker(kwargs):
     """Celery task to sync schemas."""
     call_command("sync_schemas", **kwargs)
+
+
+@shared_task
+def run_ocm_performance_in_worker(kwargs):
```

Comment:
I think we need to have the kwargs be optional for this. That way we don't need to provide the setup/teardown to the call.

Reply:
Done :+1: